### DISCUSSION TITLE
MQTT X Web Docker 1.8.3: No options for 'mqtt://' and 'mqtts://' host

### DISCUSSION THREAD

**Discussion by DatafiSystems** [emqx/MQTTX#1108] [General]
I'm not sure if this is by design but MQTT X Web Docker 1.8.3 installation only gives options for 'ws://' and 'wss://' hosts.

Mac ARM desktop installation of the MQTT X Web client gives all four ws/wss and mqtt/mqtts.

Any guidance appreciated, TIA?

**Comment by ysfscream** [comment]
MQTT X Web based on browser. Users can not use `/mqtt` to connect, only WebSocket.

Sorry for my later reply.
